package.json

```diff
@@ -107,5 +107,31 @@
     "stylelint-order": "^6.0.1",
     "vinyl": "^3.0.0",
     "wait-on": "^7.0.1"
+  },
+  "overrides": {
+    "chokidar@^2": {
+      "glob-parent": "^5"
+    },
+    "glob-stream@6": {
+      "glob-parent": "^5"
+    },
+    "latest-version@^5": {
+      "package-json": "^7"
+    },
+    "postcss-unmq": {
+      "postcss": "^7"
+    },
+    "postcss-unopacity": {
+      "postcss": "^7"
+    },
+    "postcss-unrgba": {
+      "postcss": "^7"
+    },
+    "sass-convert": {
+      "semver-regex": "^3"
+    },
+    "sassdoc-theme-default@^2": {
+      "sassdoc-extras": "^3"
```

Comment:
* https://github.com/SassDoc/sassdoc/issues/543#issuecomment-570042864

Reply:
@domoscargin I've added a better description and swapped this override

**Before:** Override `sassdoc-theme-default@2` to indirectly get `marked@0.8.2`
**After** Override `sassdoc-extras@2` to directly get `marked@0.8.2`

It's more obvious, and still resolves: https://github.com/advisories/GHSA-ch52-vgq2-943f